### PR TITLE
Add Ansible playbook to download Judgels data files, DB dump and logs

### PR DESCRIPTION
**Pull request by nathanchrs**

### CODE REVIEW

**Review by fushar — APPROVED** [review]

File: deployment/ansible/dist-example/env.yml [hunk 1]
```diff
@@ -147,3 +147,7 @@ rabbitmq_mgmt_password: xxx
 
 scoreboard_receiver_secret: xxx
 scoreboard_server_secret: xxx
+
+###################################################################################################
+
+downloaded_data_dir: "{{ inventory_dir }}/contest-data"
```
Comment (fushar) [inline deployment/ansible/dist-example/env.yml]:
```suggestion
dump_dir: "{{ inventory_dir }}/dump"
```

File: deployment/ansible/playbooks/download-data.yml [hunk 2]
```diff
@@ -0,0 +1,114 @@
+- hosts: database
```
Comment (fushar) [inline deployment/ansible/playbooks/download-data.yml]: Filename:

`dump-data.yml`

File: deployment/ansible/playbooks/download-logs.yml [hunk 3]
```diff
@@ -0,0 +1,127 @@
+- hosts: judgels
```
Comment (fushar) [inline deployment/ansible/playbooks/download-logs.yml]: Filename:

`dump-logs.yml`

**Review by fushar — CHANGES_REQUESTED** [review]

File: deployment/ansible/playbooks/download-data.yml [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+          - judgels_jerahmeel
+        target: "{{ db_dump_file }}"
+
+    - name: Download judgels DB dump file
+      synchronize:
+        mode: pull
+        src: "{{ db_dump_file }}"
+        dest: "{{ downloaded_data_dir }}/"
+
+- hosts: sandalphon
```
Comment (fushar) [inline deployment/ansible/playbooks/download-data.yml]: Please also dump submissions

File: deployment/ansible/playbooks/download-logs.yml (hunk 3 above)
Comment (nathanchrs) [reply deployment/ansible/playbooks/download-logs.yml]: OK

File: deployment/ansible/playbooks/download-data.yml (hunk 4 above)
Comment (nathanchrs) [reply deployment/ansible/playbooks/download-data.yml]: Added

**Review by fushar — APPROVED** [review]

**Pull request merged**